### PR TITLE
complete the header&headline of portfolio with linter setup #1

### PR DESCRIPTION
**Pull request by mustabbas**
complete the header&headline of a portfolio with linter setup
linter setup
add index.html file
add style.css file
complete the header&headline according to the Figma template4

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+.main-nav a {
+  display: block;
+  padding: 1em;
+  text-decoration: none;
+}
+
+.head-wraper {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
```
Comment (Murodjon000) [inline style.css]: - [ ] Please avoid using the same styling in multiple places. Stick DRY methodology in order to make your code in good shape 🙌. To see more about the DRY principle, click this [link ](https://vanseodesign.com/css/dry-principles/)👋.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+}
+
+.line {
+  min-width: 450px;
+  width: 33.333%;
+  border-bottom: 1px solid #646369;
+}
+
+.skills {
+  display: flex;
```
Comment (Murodjon000) [inline style.css:101]: - [ ] Please stick to the DRY principle, don't use one thing many times 🙏. Kindly change all duplicated things and use them one time and give them as a class to the containers.

**Review by Murodjon000 — CHANGES_REQUESTED** [review]
Hi @mustabbas, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
- [ ] Kindly fix your linter errors before making any review request. It helps you to write clean code 👏. 
### _Check the comments under the review 👇._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**